starboard/shared/modular/cobalt_layer_posix_time_abi_wrappers.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#if SB_API_VERSION >= 16
+
+#include <sys/time.h>
+#include <time.h>
+
+extern "C" {
+
+extern int __abi_wrap_clock_gettime(clockid_t clk_id, struct timespec* ts);
```

Comment:
The "extern" on the function is redundant. By default the functions have extern linkage.